sealer build e2e test failed

### What happen?

![image](https://user-images.githubusercontent.com/56665618/224294681-d379ad9e-2dc8-48b8-8465-622c6b612399.png)


### Relevant log output?

```Shell
[ERROR] [root.go:75] sealer-unknown: failed to create manifest for multi platform build: error creating image to hold manifest list: image name "***/sealerio/build-test:v1" is already associated with image "683cfcf9eb87b5cac65c2b49bd92bb35284e50644dda830d217121de3860b722": that name is already in use
```


### What you expected to happen?

_No response_

### How to reproduce it (as minimally and precisely as possible)?

_No response_

### Anything else we need to know?

_No response_

### What is the version of Sealer you using?

_No response_

### What is your OS environment?

_No response_

### What is the Kernel version?

_No response_

### Other environment you want to tell us?

- Cloud provider or hardware configuration:
- Install tools:
- Others:

--- a/test/sealer_build_test.go
+++ b/test/sealer_build_test.go
@@ -225,7 +225,7 @@ var _ = Describe("sealer build", func() {
 			testhelper.CheckErr(err)
 			testhelper.CheckExit0(sess, settings.MaxWaiteTime)
 			// check: sealer images whether image exist
-			testhelper.CheckBeTrue(build.CheckIsImageExist(imageName))
+			testhelper.CheckBeTrue(build.CheckIsMultiArchImageExist(imageName))
 
 			// check: push build image
 			testhelper.CheckErr(build.PushBuildImage(imageName))

--- a/test/suites/build/build.go
+++ b/test/suites/build/build.go
@@ -136,6 +136,12 @@ func NewArgsOfBuild() *ArgsOfBuild {
 	return &ArgsOfBuild{}
 }
 
+func CheckIsMultiArchImageExist(imageName string) bool {
+	cmd := fmt.Sprintf("%s alpha manifest inspect %s", settings.DefaultSealerBin, imageName)
+	_, err := exec.RunSimpleCmd(cmd)
+	return err == nil
+}
+
 func CheckIsImageExist(imageName string) bool {
 	cmd := fmt.Sprintf("%s inspect %s", settings.DefaultSealerBin, imageName)
 	_, err := exec.RunSimpleCmd(cmd)
@@ -157,8 +163,8 @@ func PushBuildImage(imageName string) error {
 }
 
 func DeleteBuildImage(imageName string) error {
-	push := fmt.Sprintf("%s rmi %s", settings.DefaultSealerBin, imageName)
-	_, err := exec.RunSimpleCmd(push)
+	rmi := fmt.Sprintf("%s rmi %s", settings.DefaultSealerBin, imageName)
+	_, err := exec.RunSimpleCmd(rmi)
 
 	return err
 }

